modules/logging-loki-gui-install.adoc

```diff
@@ -40,15 +40,81 @@ This option sets the `openshift.io/cluster-monitoring: "true"` label in the Name
 +
 If the approval strategy in the subscription is set to *Automatic*, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to *Manual*, you must manually approve pending updates.
 
-The Operators should now be available to all users and projects that use this cluster.
+. Create a secret.
+.. Navigate to *Workloads* -> *Secrets* in the *Administrator* perspective of the {product-title} web console.
+.. In the *Create* drop-down menu, select *From YAML*.
+.. Create a secret that uses the `access_key_id` and `access_key_secret` fields to specify your credentials and `bucketnames`, `endpoint`, and `region` to define the object storage location. AWS is used in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-s3
+  namespace: openshift-logging
+stringData:
+  access_key_id: AKIAIOSFODNN7EXAMPLE
+  access_key_secret: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  bucketnames: s3-bucket-name
+  endpoint: https://s3.eu-central-1.amazonaws.com
+  region: eu-central-1
+----
+
+. Create a `LokiStack` custom resource (CR).
+.. Navigate to *Operators* -> *Installed Operators*. Click the *All instances* tab.
+.. Use the *Create new* drop-down menu to select *LokiStack*.
+.. Select *YAML view*, then use the following template to create a `LokiStack` CR:
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: logging-loki <1>
+    namespace: openshift-logging
+  spec:
+    size: 1x.small <2>
```

Comment:
In 5.8 we introduced `1x.extra-small`? Do we want to keep `small` or produce an example with the smallest-possible configuration?

Reply:
I'd like to leave this as is for the current PR so as not to increase the scope any further, but feel free to open a Jira if this is an update you'd like to see.

Reply:
Looks like someone opened an issue for this already https://issues.redhat.com/browse/OBSDOCS-664 🙂